deploy/kubernetes/general/tools.yaml

```diff
@@ -57,6 +60,13 @@ spec:
               --name=localzk
               --hostport=zookeeper:2181
               --rootpath="/heron"
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "200M"
+            limits:
```

Comment:
Do you think imposing limits on the node could cause any unforeseen issues?

Reply:
I think I found it somewhere else and wanted to make it consistent across the files. But I do agree the limits might not be a good idea for some of the use cases such as `general` and `minikube` use cases. I'll make the updates to clean up.